design/github-actions-syncer.md

```diff
@@ -0,0 +1,78 @@
+# Conjur Synchronization
+
+## Challenge
+Github Actions can leverage Github Secrets to keep credentials secure.
+
+An organization that wants to leverage Github Actions as part of their workflow faces a serious integration challenge if they want to avoid creating security islands (credentials duplicated in multiple locations).
+
+  Actions currently lack a concept of identity. As a result, an , an organization needs to create a DAP/Conjur host (and API key). This API key needs to be stored as a GH Secret, and an SDK be written to retrieve credentials from Conjur/DAP. This marks a serious barrier to entry.
```

Comment:
typo

Reply:
shoot, that shouldn't have made it into this PR 😬.